src/steps/vulnerabilities/converters.ts

```diff
@@ -366,13 +366,16 @@ export function createVulnerabilityEntity(
         // vulnerability prioritization properties
         cvss3BaseScore: vuln.plugin.cvss3_base_score,
         cvss3TemporalScore: vuln.plugin.cvss3_temporal_score,
-
         cvssBaseScore: vuln.plugin.cvss_base_score,
         cvssTemporalScore: vuln.plugin.cvss_temporal_score,
-
         cvss3Vector: vuln.plugin.cvss3_vector?.raw,
         cvssVector: vuln.plugin.cvss_vector?.raw,
         hasPatch: vuln.plugin.has_patch,
+
+        // cross-reference plugin information array: CISA, CVE, IAVA
+        xRefs: vuln.plugin.xrefs
+          ? vuln.plugin.xrefs.map((r) => JSON.stringify(r))
+          : [],
```

Comment:
I got a notification by mail, but this is better :)

```suggestion
        xRefs: vuln.plugin.xrefs?.map(JSON.stringify) ?? [];
```

Reply:
thank you Nacho! I'm stuck here with the new line

<img width="1062" alt="image" src="https://github.com/JupiterOne/graph-tenable-io/assets/6811855/c99087c7-3dce-4c75-90b5-00dae9283b6e">

Reply:
Hmm, just try with your approach and add ?? in case xrefs is undefined.

`vuln.plugin.xrefs?.map(r => JSON.stringify(r)) ?? [];`

Reply:
ok updated to the one liner with a comma on the end